### PR TITLE
feature/128/api

### PR DESCRIPTION
**Pull request by theapache64**
Continuation of #267, Part of implementing #128

### CODE REVIEW

File: api/src/main/kotlin/wu/seal/jsontokotlinclass/server/controllers/AddHitController.kt [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            "To add new analytics event"
+    )
+    @PostMapping("/add_hit")
+    @ResponseBody
+    fun addHit(@RequestBody addHitRequest: AddHitRequest): AddHitResponse {
+
+        println("Hit!")
+
+        verifyClient(addHitRequest.client)
+        hitsRepo.save(
```
Comment (wuseal) [inline api/src/main/kotlin/wu/seal/jsontokotlinclass/server/controllers/AddHitController.kt:44]: Nice Code 👍

File: api/src/main/kotlin/wu/seal/jsontokotlinclass/server/models/routes/addhit/AddHitResponse.kt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package wu.seal.jsontokotlinclass.server.models.routes.addhit
+
+import wu.seal.jsontokotlinclass.server.models.routes.base.BaseResponse
+
+class AddHitResponse(data: Void?, error: Boolean, errorCode: Int, message: String) : BaseResponse<Void>(data, error, errorCode, message) {
```
Comment (wuseal) [inline api/src/main/kotlin/wu/seal/jsontokotlinclass/server/models/routes/addhit/AddHitResponse.kt:5]: Data Type is `Void`? Hey, what's the meaning?

Comment (theapache64) [reply api/src/main/kotlin/wu/seal/jsontokotlinclass/server/models/routes/addhit/AddHitResponse.kt:5]: As adding a hit is simply an insertion event, the `data` in the response `JSON`  from the `addHit` route will be null.

**Comment by theapache64** [comment]
Thank you Seal

File: api/src/main/kotlin/wu/seal/jsontokotlinclass/server/models/routes/addhit/AddHitResponse.kt (hunk 2 above)
Comment (wuseal) [reply api/src/main/kotlin/wu/seal/jsontokotlinclass/server/models/routes/addhit/AddHitResponse.kt:5]: Sure! 😄

**Pull request merged**